brewchat/src/main/java/com/example/brewchat/Application.java

```diff
@@ -42,6 +42,7 @@ public void onCreate() {
                 getString(R.string.app_id),
                 getString(R.string.auth_key),
                 getString(R.string.auth_secret));
+        chatService.createSession();
```

Comment:
I didn't ask the right question... I think that QB server gives you a token when you open a session and this token should be renewed at some point.

So why not try to create a new session every time that the user what to register or login?